README.md

```diff
@@ -1 +1,35 @@
-# CarbonQueryDatabase
\ No newline at end of file
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/11a3ucotxcl9746k/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/robot-toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Robot_Toolkit/Robot_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=93&branchName=master)
```

Comment:
References to `robot-toolkit` should be changed to reference this toolkit.

Reply:
Updated, the link to https://ci.appveyor.com/project/BHoMBot/CarbonQueryDatabase-Toolkit/branch/master appears to be a dead page unlike the Robot one, how do I get that page up and running for the build check on master?

Reply:
As discussed @enarhi the page will be set up shortly so can go ahead and use it for now and accept the dead page :smile: